src/components/tabs/hooks/use-tab-items.ts

```diff
@@ -160,8 +160,16 @@ function validateTabChildren(
 	 * each of which represent a valid tab item to render
 	 */
 	const tabItems = childrenArray.map((childTab: ReactElement) => {
-		const { children, group, heading, icon } = childTab.props
-		return { content: children, group, icon, label: heading }
+		const { children, renderContent, group, heading, headingSlot, icon } =
+			childTab.props
+		return {
+			content: children,
+			renderContent,
+			group,
+			icon,
+			label: heading,
+			labelSlot: headingSlot,
```

Comment:
❓ should we check for either `label` or `labelSlot`, but not both? Definitely low prio, unless we use the `label` as well even when using `labelSlot`.

Reply:
✅ I've added comments to `types` files to clarify the relationship between `label` and `labelSlot`.